test/AWS.Deploy.CLI.Common.UnitTests/Recipes/Validation/ECSFargateOptionSettingItemValidationTests.cs

```diff
@@ -43,7 +43,7 @@ public ECSFargateOptionSettingItemValidationTests()
             _optionSettingHandler = new OptionSettingHandler(new ValidatorFactory(_serviceProvider));
 
             _recipe = new RecipeDefinition("Fargate", "0.1", "Fargate", DeploymentTypes.CdkProject, DeploymentBundleTypes.Container, "", "", "", "", "");
-            _recommendation = new Recommendation(_recipe, null, 100, new Dictionary<string, object>());
+            _recommendation = new Recommendation(_recipe, null!, 100, new Dictionary<string, object>());
```

Comment:
is `null!` a valid thing to do?

Reply:
This is used when we want to force a value to be null. It's fine to be used in certain cases. Also in the test suite it makes sense to avoid refactoring a lot and the tests would fail if anything unexpected happened.